### PR TITLE
data: sri-rishi

### PR DESCRIPTION
**Pull request by sri-rishi**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4494"></a>

### CODE REVIEW

**Review by priyankarpal — APPROVED** [review]
LGTM  👍🏻

File: RishiSrivastava.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		},
+		{
+			"name": "Follow me on Github",
+			"url": "https://github.com/sri-rishi",
+			"icon": "FaGithub"
+		},
+    {
+			"name": "Follow me on Hashnode",
+			"url": "https://hashnode.com/@sririshi",
+			"icon": "FaHashnode"
```
Comment (kumarsonsoff3) [inline RishiSrivastava.json:24]: You probably be using an invalid icon here. It should be `SiHashnode`.

```suggestion
			"icon": "SiHashnode"
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Your file should be in `data` directory, and filename should be exact same as your username. So please rename your [**file here**](https://github.com/sri-rishi/LinkFree/edit/sri-rishi-patch-2/RishiSrivastava.json) to `data/sri-rishi.json`.

Change `RishiSrivastava.json`  - > `data/sri-rishi.json` here. 

Let us know if you need any help with this! 

image

**Review by priyankarpal — CHANGES_REQUESTED** [review]
You need to fix the code

File: RishiSrivastava.json (hunk 1 above)
Comment (sri-rishi) [reply RishiSrivastava.json]: As mentioned I have made changes

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**